scripts/variantstore/wdl/GvsExtractCohortFromSampleNames.wdl

```diff
@@ -34,10 +35,22 @@ workflow GvsExtractCohortFromSampleNames {
     File? gatk_override
   }
 
+  # writing the array to a file has to be done in a task
+  # https://support.terra.bio/hc/en-us/community/posts/360071465631-write-lines-write-map-write-tsv-write-json-fail-when-run-in-a-workflow-rather-than-in-a-task
```

Comment:
uh okay, TIL 🙂

Reply:
yeah -- def a bummer, but seems to be the way it is